etl/steps/data/garden/wb/2025-08-04/food_prices_for_nutrition.meta.yml

```diff
@@ -0,0 +1,443 @@
+definitions:
+  common:
+    processing_level: minor
+    presentation:
+      topic_tags:
+        - Food Prices
+        - Poverty
+  description_key_pct_of_population: &description_key_pct_of_population
+    # Original description:
+    # The indicator expresses the percentage of the total population unable to afford an energy sufficient diet in a given country and year. A diet is considered unaffordable when the diet cost plus expenditures for basic non-food needs exceed income per capita per day. The expenditures for basic non-food needs are calculated as average non-food expenditure shares of low-income consumer multipled by internaitonal poverty lines set by the World Bank. The non-food expenditure share is 37% and 44% in low-income and lower-middle-income countries for the second quintile of consumers, and 54% in upper-middle-and-high-income countries for the first quintile of consumers, according to household surveys compiled by the World Bank. The international poverty lines are $2.15/day for low-income countries, $3.65/day for lower-middle-income countries, $6.85/day for upper-middle-income countries, and $24.36/day for high-income countries, in 2017PPP$. Countries' income classifications follow the calendar year of 2021 standard (fiscal year of 2023 of the World Bank), which is the base year of the latest ICP cycle. Income data are provided by the World Bank’s Poverty and Inequality Platform. A value of zero indicates a null or a small number rounded down at the current precision level.
+    - Non-food expenses are estimated by looking at how much low-income people typically spend on things like housing, clothing, and transportation.
+    - This amount is calculated as the average share of non-food expenditure for a low-income consumer, multiplied by the international poverty line.
+    - "The average share of non-food expenditure is based on the spending habits of specific income groups (quintiles):
+
+      - In upper-middle-income and high-income countries, the first quintile of consumers is considered (the poorest 20% of consumers), where people spend around 54% of their income on non-food needs.
+
+      - In low-income and lower-middle-income countries, the second quintile is considered (the second lowest 20% of consumers), where people spend about 37% and 44% of their income on non-food needs, respectively."
+    - "The international poverty lines (in [international-$](#dod:int_dollar_abbreviation) at 2017 prices) are:
+
+      - For low-income countries: $2.15/day.
```

Comment:
Not sure if all these descriptions need to be updated?
I understand that the dataset is now using 2021 PPPs. If so, the lines are
3, 4.20, 8.30, and 29.40